### ISSUE TITLE
Group does not do anything in entity card

### ISSUE THREAD

**Issue by KennethLavrsen** [berrywhite96/lovelace-shutter-row#23]
I added the group: true to a shutter row
I cannot see any difference. True or false there is still a frame around

![image](https://user-images.githubusercontent.com/8373419/225109625-ed37e684-85a5-4f3d-bb43-33bec5d8900b.png)

```
  - type: custom:shutter-row
    entity: cover.cloud
    group: true
    invert_position_label: true
    state_color: true
```

**Comment by berrywhite96** [comment]
Please check if you are on 0.3.0 or higher.

1. Call HA on desktop PC in Chrome
2. Press F12 and click on `Console`
3. There should be a yellow log entry with the loaded shutter row version.

**Comment by KennethLavrsen** [comment]
This is really odd

I had already checked that the installed version inside HACS was 0.3.2
And then I went to the actual JS files and checked that the version was 0.3.2

But your posting using F12 and console revealed that the browser had a much older version.

I have done control F5 multiple times. And I have had the same symptom on multiple computers. They cache this JS file in a way that does not get refreshed.

What helped was to put http://192.168.1.X:8123/hacsfiles/lovelace-shutter-row/shutter-row.js?hacstag=524783308020 in the browser URL to force it to cache update.  I wonder how many old JS files people are fighting with and reporting bogus error reports from. You can normally trust a control F5 to refresh all JS used in a given browser URL

Once I did this it sort of works. I have no border

But the text is poorly alligned with the other entities and my icon is not shown (it is not a default shutter icon)

![image](https://user-images.githubusercontent.com/8373419/225130244-20009129-c49c-4b95-ada4-c98fb67778fc.png)

**Comment by berrywhite96** [comment]
Can you post the attribute content from the developer tools?

**Comment by KennethLavrsen** [comment]
I do not understand that question.

**Comment by KennethLavrsen** [comment]
2 minutes later. Oh now I know what you mean. For the entity I show sure

It is an MQTT device called a Cloud. A small long narrow curtain that I can move so it covers the sun. So I call it the cloud

```
current_position: 100
icon: mdi:cloud
friendly_name: Cloud
supported_features: 15
```

**Comment by berrywhite96** [comment]
Go to this url: https://homeassistant.local:8123/developer-tools/state
Search for your cover entity and copy the content from the 'Attributes' column and post it here.

**Comment by KennethLavrsen** [comment]
Yes. Got it above. I was in the F12 content mode and could not find any attributes. And then I got the "DUH! - me idiot" moment a minute after I posted

**Comment by berrywhite96** [comment]
Never mind this is another bug that happens on custom entity icon...

Will fix that this week!

**Comment by KennethLavrsen** [comment]
Thanks for following up.

**Comment by berrywhite96** [comment]
Fixed with v0.3.3.
